tests/test_units.py

```diff
@@ -0,0 +1,108 @@
+from rdflib import Namespace
+import re
+import brickschema
+from collections import defaultdict
+import sys
+
+sys.path.append("..")
+from bricksrc.namespaces import A, BRICK, TAG, QUDT  # noqa: E402
+
+BLDG = Namespace("https://brickschema.org/schema/ExampleBuilding#")
+
+
+def test_quantity_has_one_quantitykind():
+    """
+    In the current implementation, using owl:equivalentClass to align Quantity
+    with QUDT QuantityKinds, we need to make sure that through RDFS subclassing
+    that a Brick Quantity does not end up with more than 1 QuantityKind
+    """
+    g = brickschema.graph.Graph()
+    g.load_file("Brick.ttl")
+    g.g.bind("qudt", QUDT)
+    g = brickschema.inference.OWLRLInferenceSession(load_brick=False).expand(g)
+    quantity_qk = g.query(
+        "SELECT ?quantity ?kind WHERE {\
+            ?quantity   a   brick:Quantity .\
+            ?quantity   owl:sameAs ?kind }"
+    )
+    assert len(quantity_qk) > 0
+    seen = defaultdict(list)
+    for quant, quantkind in quantity_qk:
+        if quant == quantkind:
+            continue
+        if "Brick" in quant and "qudt" in quantkind:
+            seen[quant].append(quantkind)
+    for quant, kindlist in seen.items():
+        assert (
+            len(kindlist) == 1
+        ), f"Quantity {quant} has more than one associated QuantityKind! {kindlist}"
+
+
+def test_instances_measure_correct_units():
```

Comment:
Could you clarify what this test does once more? Instance--hasUnit-->qudt:Unit is generated from the class--...-->qudt:Unit, so you are comparing A to B while A is logically equivalent to B in the generation code. Could you give an example of what kind of error this can detect?

Reply:
Good catch! This test is a carry over from when it was class-based, so it's not  verifying anything anymore

Reply:
I guess it provides an example of how `hasUnit` is used with Point instances -- I'll keep the implementation for now so we can build it into something more useful later